### DISCUSSION TITLE
How do i set power on laser

### DISCUSSION THREAD

**Discussion by George162** [arkypita/LaserGRBL#1316] [Q&A]

**Comment by StuartB4** [comment]
The maximum power of the laser is set in Grbl configuration on the Grbl menu. 

Start lasergrbl and click connect. Goto Grbl menu and click grbl configuration. 

Make sure $32 is to 1. That lets the PWM signal control the power from 0(off) to full power.
Set $30 to 255. then click Write to store the settings.

Then when you want to adjust the power in LaserGRBL you can change the setting in S-MAX in Target image settings. 
S-MIN is the minimum setting. So anywhere between S-MIN and S-MAX is the power level.
